schemas/opticalData.json

```diff
@@ -268,27 +268,15 @@
               "properties": {
                 "polar": {
                   "$ref": "number.json#/$defs/degree",
-                  "description": "The polar angle is the angle between the direction of incidence and an axis which is perpendicular to the surface of the sample. A polar angle of 0° means that the direction of incidence is perpendicular to the sample."
+                  "description": "The polar angle is the angle between the direction of incidence and an axis which is perpendicular to the surface of the sample. A polar angle of 0° means that the direction of incidence is perpendicular to the sample. If the sample has rotational symmetry, then the polar angle is sufficient to define the direction of incidence."
```

Comment:
If the sample has an `incidenceAngle` symmetry, then ...

Reply:
Does https://github.com/ise621/building-envelope-data/pull/156/commits/a333c349503c6d56d0a88c37f080ee1efaf67334 resolve that? I'm unsure why `rotational` should be replaced by `incidence-angle` because `rotational` was used before for the property `incidence` and I just copied and pasted the description.

Reply:
Yes, a333c34 a333c349503c6d56d0a88c37f080ee1efaf67334 [a333c34](https://github.com/ise621/building-envelope-data/commit/a333c349503c6d56d0a88c37f080ee1efaf67334) resolves that. 

A [rotational symmetry](https://github.com/ise621/building-envelope-data/blob/badc2d264011e381a2b9350a1d40ea41fb5c7c12/schemas/opticalData.json#L170) means that for example one quarter of the hemisphere is enough to describe the whole hemisphere because the optical results for the other quarters is identical. An incidenceAngle symmetry means that the the optical results are the same for all polar angles. All samples with incidenceAngle symmetry have also a rotational symmetry, but samples with rotational symmetry have different optical results for different directions which have the same polar angle. Several scientists say rotational and but mean incidenceAngle symmetry.

@HelenRoseWilson Please confirm the last paragraph. Or is there a better, more intuitiv wording in order to avoid mistakes?

Reply:
@HelenRoseWilson and me agree that:

- A `rotational` symmetry means that for example one quarter of the hemisphere is enough to describe the whole hemisphere because the optical results for the other quarters can be obtained by rotating the results for the first quarter around the axis perpendicular to the sample. However, if the optical results depend only on the `polar` angle and not on the `azimuth` angle, use `azimuthalAngleInvariance` instead of `rotational`!
- An `azimuthalAngleInvariance` means that the optical results depend only on the `polar` angle and not on the `azimuth` angle. Many homogeneous materials like glass panes have an `azimuthalAngleInvariance`.
- `azimuthalAngleInvariance` is therefore a special case of a `rotational` symmetry. This means that all samples with `azimuthalAngleInvariance` have also a `rotational` symmetry, but not all samples with `rotational` symmetry have a `azimuthalAngleInvariance`. 

To do:

- [ ] Update the [symmetry](https://github.com/ise621/building-envelope-data/blob/6bfcde0becb7797b633c6be16424f408e0bd5e29/schemas/opticalData.json#L206) accordingly
- [ ] Substitute `incidenceAngle` in the entire repository by `azimuthalAngleInvariance`